backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/ProprietaryService.java

```diff
@@ -102,9 +110,12 @@ public Proprietary updateProprietaryFrameFromNorm(
       query.metadata().organisationsEinheit()
     );
 
-    updateNormPort.updateNorm(new UpdateNormPort.Command(norm));
+    Map<ExpressionEli, Norm> updatedNorms = normService.updateNorm(norm);
 
-    return proprietary;
+    Norm updatedNorm = updatedNorms.get(query.eli());
+    if (updatedNorm != null) return updatedNorm
+      .getMeta()
+      .getOrCreateProprietary(); else return null;
```

Comment:
I find it a bit hard to read this shorthand notation when used over multiple lines